ansible_collections/arista/avd/roles/eos_designs/python_modules/mlag/avdstructuredconfig.py

```diff
@@ -337,10 +385,10 @@ def router_bgp(self):
         if self._mlag_ibgp_origin_incomplete is True:
             peer_group["route_map_in"] = "RM-MLAG-PEER-IN"
 
-        router_bgp["peer_groups"] = {peer_group_name: peer_group}
+        router_bgp_cfg["peer_groups"] = {peer_group_name: peer_group}
```

Comment:
in term of consistency between method I am not sure why we use in one place `router_bgp` and in the other `router_bgp_cfg`?

Reply:
fixed